### PR TITLE
feature:optimize config code

### PR DESCRIPTION
**Pull request by reckless11**
1.get configuration api directly get data from client
2.handle exception case,throw the exception in init process and catch the exception in later update process
3.add log for config module

# Description

_Please explain the changes you've made_

## Issue reference

We strive to have all PR being opened based on an issue, where the problem or feature have been discussed prior to implementation.

Please reference the issue this PR will close: #34 

## Checklist

Please make sure you've completed the relevant tasks for this PR, out of the following list:

* [ √] Code compiles correctly
* [ √] Created/updated tests
* [ √] Extended the documentation

### CODE REVIEW

File: capa-spi-aws-config/src/main/java/group/rxcloud/capa/spi/aws/config/AwsCapaConfigStore.java [hunk 1]
```diff
@@ -103,57 +109,42 @@ public void close() {
...
+     * @param <T>      T
+     * @return config value or throw exception if cannot get config in specific time.
+     * Exception contains e.g. {@link BadRequestException},{@link TimeoutException} and so on.
+     */
     @Override
     protected <T> Mono<List<ConfigurationItem<T>>> doGet(String appId, String group, String label, List<String> keys, Map<String, String> metadata, TypeRef<T> type) {
         List<ConfigurationItem<T>> items = new ArrayList<>();
         if (CollectionUtils.isNullOrEmpty(keys)) {
-            return Mono.error(new IllegalArgumentException("keys is null or empty"));
+            LOGGER.warn("keys is null or empty,appId:{}", appId);
```
Comment (kevinten10) [inline capa-spi-aws-config/src/main/java/group/rxcloud/capa/spi/aws/config/AwsCapaConfigStore.java:129]: log添加一些前缀吧，方便识别到是哪个功能的warn。
例如加一个[Capa.Configuration]

File: capa-spi-aws-config/src/main/java/group/rxcloud/capa/spi/aws/config/constants/AwsConfigConstants.java [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+ * limitations under the License.
+ */
+package group.rxcloud.capa.spi.aws.config.constants;
+
+/**
+ * @author Reckless Xu
+ * 2021/12/1
+ */
+public interface AwsConfigConstants {
+    long REQUEST_TIMEOUT_IN_SECONDS = 15L;
```
Comment (kevinten10) [inline capa-spi-aws-config/src/main/java/group/rxcloud/capa/spi/aws/config/constants/AwsConfigConstants.java:24]: 这个配置可以放到 capa-component-configuration-aws.properties 文件中么，目前应该有一个统一的工具类可以获取这里面的值

**Review by kevinten10 — APPROVED** [review]
LGTM.

File: capa-spi-aws-config/src/main/java/group/rxcloud/capa/spi/aws/config/AwsCapaConfigStore.java (hunk 1 above)
Comment (reckless11) [reply capa-spi-aws-config/src/main/java/group/rxcloud/capa/spi/aws/config/AwsCapaConfigStore.java]: 已添加

File: capa-spi-aws-config/src/main/java/group/rxcloud/capa/spi/aws/config/constants/AwsConfigConstants.java (hunk 2 above)
Comment (reckless11) [reply capa-spi-aws-config/src/main/java/group/rxcloud/capa/spi/aws/config/constants/AwsConfigConstants.java]: 已将timeout归入capa-component-configuration-aws.properties文件

**Pull request merged**